app/assets/stylesheets/ama_layout/layout_components/cart.scss

```diff
@@ -0,0 +1,145 @@
+/*Cart*/
+.summary-cart.active.summary_height{
+  min-height: 300px;
+}
+
+.summary-cart{
+  ::-webkit-scrollbar{
+    -webkit-appearance: none;
+    width: 7px;
+  }
+
+  ::-webkit-scrollbar-thumb{
+    border-radius: 4px;
+    background-color: rgba(0, 0, 0, .5);
+    -webkit-box-shadow: 0 0 1px rgba(255, 255, 255, .5);
+  }
+
+  .summary-cart-body{
+    background-color: $stone;
+    padding: $base-padding*2;
+    font-size: $base-font-size*0.875;
+    color: $charcoal;
+
+    ul{
+      list-style: none;
+      margin: 0;
+
+      li{
+        padding: 8px;
+      }
+
+      li:not(:nth-child(1)){
+        border-top: 1px solid $smoke;
+      }
+
+      .summary-flash-notice{
+        color: $white;
+        background: $brand-blue-insurance;
+
+        p,
+        a,
+        a:hover,
+        a:focus{
+          color: $white;
+        }
+
+        &:not(:nth-child(1)){
+          border-top: none;
+        }
+      }
+    }
+
+    .items{
+      overflow-y: scroll;
+      max-height: auto;
+    }
+
+    .items_many_mobile{
```

Comment:
I'll drop it as part of this PR once amaabca/registries#97 goes in. Mostly because it's a real pain to bundle update ama_layout in allthethings, so we might as well just do it once tomorrow.